Wazuh dashboard workflow run fails when managing the base file

# Description

Testing the release package generation script I have found an error building the Wazuh dashboard:

```
Run mv /home/runner/work/wazuh-dashboard/wazuh-dashboard/artifacts/target/opensearch-dashboards-2.13.0-linux-x64.tar.gz /home/runner/work/wazuh-dashboard/wazuh-dashboard/artifacts/target/wazuh-dashboard_4.9.0-00_x64_enhancement/wj6371-add-workflow-id.tar.gz
  mv /home/runner/work/wazuh-dashboard/wazuh-dashboard/artifacts/target/opensearch-dashboards-2.13.0-linux-x64.tar.gz /home/runner/work/wazuh-dashboard/wazuh-dashboard/artifacts/target/wazuh-dashboard_4.9.0-00_x64_enhancement/wj6371-add-workflow-id.tar.gz
  shell: /usr/bin/bash -e {0}
  env:
    NPM_CONFIG_USERCONFIG: /home/runner/work/_temp/.npmrc
    NODE_AUTH_TOKEN: XXXXX-XXXXX-XXXXX-XXXXX
    YARN_CACHE_LOCATION: /home/runner/.cache/yarn/v6
    VERSION: 2.13.0
    WZD_VERSION: 4.9.0
    WZD_REVISION: 00
    ARTIFACT_BUILD_NAME: wazuh-dashboard_4.9.0-00_x64_enhancement/wj6371-add-workflow-id.tar.gz
mv: cannot move '/home/runner/work/wazuh-dashboard/wazuh-dashboard/artifacts/target/opensearch-dashboards-2.13.0-linux-x64.tar.gz' to '/home/runner/work/wazuh-dashboard/wazuh-dashboard/artifacts/target/wazuh-dashboard_4.9.0-00_x64_enhancement/wj6371-add-workflow-id.tar.gz': No such file or directory
Error: Process completed with exit code 1.
```


- https://github.com/wazuh/wazuh-dashboard/actions/runs/9117577629

I used the following parameters:

```
{'url': 'https://api.github.com/repos/wazuh/wazuh-dashboard/actions/workflows/build_wazuh_dashboard_with_plugins.yml/dispatches', 'headers': {'Authorization': 'token xxxxx', 'Accept': 'application/vnd.github.v3+json'}, 'payload': {'ref': 'enhancement/wj6371-add-workflow-id', 'inputs': {'is_stage': True, 'checksum': True, 'system': 'deb', 'revision': '1', 'architecture': 'amd64', 'reference_security_plugins': '4.9.0', 'reference_wazuh_plugins': '4.9.0'}}}`
```

Note that I used the revision parameter with value: `1`, but the base file is being called as `00`, I don't know if this is related or not, but we should review it

```
WZD_REVISION: 00
wazuh-dashboard_4.9.0-00_x64_enhancement/wj6371-add-workflow-id.tar.gz
```

## Related

- https://github.com/wazuh/wazuh-jenkins/issues/6371#issuecomment-2115914391

--- a/.github/workflows/build_and_test_workflow.yml
+++ b/.github/workflows/build_and_test_workflow.yml
@@ -139,7 +139,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest] #Removed 'windows-latest'
-        group: [1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 13]
+        group: [1, 2, 4, 6, 7, 8, 9, 10, 11, 13]
         include:
           - os: ubuntu-latest
             name: Linux

--- a/.github/workflows/build_base.yml
+++ b/.github/workflows/build_base.yml
@@ -27,8 +27,8 @@ jobs:
         working-directory: ./artifacts
     strategy:
       matrix:
-        DISTRIBUTION: [ tar.gz ]
-        ARCHITECTURE: [ x64 ]
+        DISTRIBUTION: [tar.gz]
+        ARCHITECTURE: [x64]
 
     steps:
       - name: Checkout code
@@ -69,7 +69,7 @@ jobs:
 
       - name: Get artifact build name
         run: |
-          echo "ARTIFACT_BUILD_NAME=wazuh-dashboard_${{ env.WZD_VERSION }}-${{ env.WZD_REVISION }}_${{ matrix.ARCHITECTURE }}_${{ inputs.CHECKOUT_TO }}.${{ matrix.DISTRIBUTION }}" >> $GITHUB_ENV
+          echo "ARTIFACT_BUILD_NAME=wazuh-dashboard_${{ env.WZD_VERSION }}-${{ env.WZD_REVISION }}_${{ matrix.ARCHITECTURE }}.${{ matrix.DISTRIBUTION }}" >> $GITHUB_ENV
 
       - name: Run bootstrap
         run: yarn osd bootstrap

--- a/.github/workflows/build_wazuh_dashboard_with_plugins.yml
+++ b/.github/workflows/build_wazuh_dashboard_with_plugins.yml
@@ -45,7 +45,7 @@ on:
         required: true
         default: false
       id:
-        description: "ID used to identify the workflow uniquely."
+        description: 'ID used to identify the workflow uniquely.'
         type: string
         required: false
 
@@ -146,7 +146,7 @@ jobs:
 
       - name: Setup packages names
         run: |
-          echo "WAZUH_DASHBOARD_SLIM=wazuh-dashboard_${{ env.VERSION }}-${{ env.REVISION }}_x64_${{ github.head_ref || github.ref_name }}.tar.gz" >> $GITHUB_ENV
+          echo "WAZUH_DASHBOARD_SLIM=wazuh-dashboard_${{ env.VERSION }}-${{ env.REVISION }}_x64_.tar.gz" >> $GITHUB_ENV
           echo "WAZUH_SECURITY_PLUGIN=wazuh-security-dashboards-plugin_${{ env.VERSION }}-${{ env.REVISION }}_${{ inputs.reference_security_plugins }}.zip" >> $GITHUB_ENV
           echo "WAZUH_PLUGINS=wazuh-dashboard-plugins_${{ env.VERSION }}-${{ env.REVISION }}_${{ inputs.reference_wazuh_plugins }}.zip" >> $GITHUB_ENV
           if [ "${{ inputs.system }}" = "deb" ]; then

--- a/.github/workflows/build_wazuh_dashboard_with_plugins.yml
+++ b/.github/workflows/build_wazuh_dashboard_with_plugins.yml
@@ -102,7 +102,7 @@ jobs:
   build-base:
     needs: [validate-inputs]
     name: Build dashboard
-    uses: wazuh/wazuh-dashboard/.github/workflows/build_base.yml@bug/package-generation-branch-with-slash
+    uses: wazuh/wazuh-dashboard/.github/workflows/build_base.yml@4.9.0
     with:
       CHECKOUT_TO: ${{ github.head_ref || github.ref_name }}
 

--- a/.github/workflows/build_wazuh_dashboard_with_plugins.yml
+++ b/.github/workflows/build_wazuh_dashboard_with_plugins.yml
@@ -146,7 +146,7 @@ jobs:
 
       - name: Setup packages names
         run: |
-          echo "WAZUH_DASHBOARD_SLIM=wazuh-dashboard_${{ env.VERSION }}-${{ env.REVISION }}_x64_.tar.gz" >> $GITHUB_ENV
+          echo "WAZUH_DASHBOARD_SLIM=wazuh-dashboard_${{ env.VERSION }}-${{ env.REVISION }}_x64.tar.gz" >> $GITHUB_ENV
           echo "WAZUH_SECURITY_PLUGIN=wazuh-security-dashboards-plugin_${{ env.VERSION }}-${{ env.REVISION }}_${{ inputs.reference_security_plugins }}.zip" >> $GITHUB_ENV
           echo "WAZUH_PLUGINS=wazuh-dashboard-plugins_${{ env.VERSION }}-${{ env.REVISION }}_${{ inputs.reference_wazuh_plugins }}.zip" >> $GITHUB_ENV
           if [ "${{ inputs.system }}" = "deb" ]; then

--- a/.github/workflows/build_wazuh_dashboard_with_plugins.yml
+++ b/.github/workflows/build_wazuh_dashboard_with_plugins.yml
@@ -102,7 +102,7 @@ jobs:
   build-base:
     needs: [validate-inputs]
     name: Build dashboard
-    uses: wazuh/wazuh-dashboard/.github/workflows/build_base.yml@4.9.0
+    uses: wazuh/wazuh-dashboard/.github/workflows/build_base.yml@bug/package-generation-branch-with-slash
     with:
       CHECKOUT_TO: ${{ github.head_ref || github.ref_name }}
 

--- a/dev-tools/test-packages/deb/Dockerfile
+++ b/dev-tools/test-packages/deb/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:latest
+FROM ubuntu:jammy
 ARG PACKAGE
 RUN mkdir -p /tmp
 RUN apt-get update --fix-missing

